scripts/erc20_to_erc20/sendHome.js

```diff
@@ -80,7 +80,7 @@ async function main() {
         privateKey: USER_ADDRESS_PRIVATE_KEY,
         data,
         nonce,
-        gasPrice: '1',
+        gasPrice: '1000000000',
```

Comment:
Can we introduce a parameter to configure the gas price which will be used for testing transactions?

Reply:
Done. Added `HOME_TEST_TX_GAS_PRICE` and `FOREIGN_TEST_TX_GAS_PRICE`